cmake/projects/Lua/hunter.cmake

```diff
@@ -8,6 +8,17 @@ include(hunter_pick_scheme)
 include(hunter_cacheable)
 include(hunter_configuration_types)
 
+hunter_add_version(
+    PACKAGE_NAME
+    Lua
+    VERSION
+    "5.3.5"
+    URL
+    "https://github.com/cpp-pm/lua/archive/v5.3.5-p0.tar.gz"
```

Comment:
The PR to cpp-pm/lua isn’t up yet right?

Reply:
The PR is there now, however previously I checked in the changes to a 5.3.5-p0 branch and released from there.